client/src/components/goals-list/goals-list.js

```diff
@@ -1,14 +1,34 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Goal from '../goal';
+import KeyListener from '../key-listener';
 
-const GoalsList = ({ goals }) => (
-  <div>
-    {goals.map(goal => (
-      <Goal key={goal.id} goal={goal} />
-    ))}
-  </div>
-);
+class GoalsList extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      showStoryOwners: false,
```

Comment:
For the sidebar I persisted the open/close state in local storage so that it persists when reloading the page or switching teams.

https://github.com/geckoboard/keeper/blob/master/client/src/components/sidebar/sidebar.js#L9-L15

I think that would also make sense to do here.